.changeset/old-stingrays-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-site': minor
+---
+
+Added a CSP to both standard and token pages
```

Comment:
nit: This sentence will show up in the change log, so it's best to add a period at the end to make it a proper sentence.